crates/storage/db-api/src/models/mod.rs

```diff
@@ -373,7 +372,6 @@ mod tests {
         assert_eq!(StoredBlockOmmers::bitflag_encoded_bytes(), 0);
         assert_eq!(StoredBlockWithdrawals::bitflag_encoded_bytes(), 0);
         assert_eq!(StorageHashingCheckpoint::bitflag_encoded_bytes(), 1);
-        assert_eq!(TxEip2930::bitflag_encoded_bytes(), 3);
```

Comment:
we should keep this test, for now we're accumulating them here:
https://github.com/paradigmxyz/reth/blob/7269cf28538d25d87bd85bb3594dc8616d674370/crates/storage/codecs/src/alloy/transaction/mod.rs#L16-L19

Reply:
fixed